conan/api/subapi/workspace.py

```diff
@@ -201,6 +203,45 @@ def add(self, path, name=None, version=None, user=None, channel=None, cwd=None,
         self._ws.add(ref, full_path, output_folder, product)
         return ref
 
+    @staticmethod
+    def initialize(path):
+        abs_path = make_abs_path(path)
+        os.makedirs(abs_path, exist_ok=True)
+        ws_yml_file = Path(abs_path, WORKSPACE_YML)
+        ws_py_file = Path(abs_path, WORKSPACE_PY)
+        if not ws_yml_file.exists():
+            ConanOutput().success(f"Created empty {WORKSPACE_YML} in {path}")
+            save(ws_yml_file, "")
+        if not ws_py_file.exists():
+            ConanOutput().success(f"Created empty {WORKSPACE_PY} in {path}")
+            save(ws_py_file, textwrap.dedent('''\
+            from conan import Workspace
+            from conan import ConanFile
+            from conan.tools.cmake import CMakeDeps, CMakeToolchain, cmake_layout
+
+
+            class MyWs(ConanFile):
+                """ This is a special conanfile, used only for workspace definition of layout
+                and generators. It shouldn't have requirements, tool_requirements. It shouldn't have
+                build() or package() methods
+                """
+                settings = "os", "compiler", "build_type", "arch"
+
+                def generate(self):
+                    deps = CMakeDeps(self)
+                    deps.generate()
+                    tc = CMakeToolchain(self)
+                    tc.generate()
+
+                def layout(self):
+                    cmake_layout(self)
+
+
+            class Ws(Workspace):
+                def root_conanfile(self):
+                    return MyWs
+            '''))
```

Comment:
Should I import the `conanws.py` from the `new` module? Should I change this? Simpler?